.circleci/config.yml

```diff
@@ -2,30 +2,27 @@ version: 2
 jobs:
   build:
     machine:
-      enabled: true
+      image: ubuntu-1604:201903-01
     steps:
       - checkout
-      - run:
-          name: Copy env file
-          command: cp .env.dist .env
-      - run:
-          name: Setup environment
-          command: ./bin/docker/pim-setup.sh
       - run:
           name: Change owner on project dir (default user = circleci (1001) and docker needs uid 1000)
           command: sudo chown -R 1000:1000 ../project
       - run:
-          name: Start containers
-          command: docker-compose up -d
+          name: Install PHP dependencies
+          command: docker run -u www-data -u www-data -v $(pwd):/srv/pim -w /srv/pim --rm akeneo/pim-php-dev:4.0 php -d memory_limit=4G /usr/local/bin/composer install
```

Comment:
make vendor?

Reply:
Nope, there's no makefile at this point (it will comes from the pim-community-dev).